specs/grafana-integration.adoc

```diff
@@ -0,0 +1,140 @@
+// vim: tw=79
+
+= Packaging tendrl grafana integration and changes
+
+Grafana is a monitoring tool works with some standard time-series db which is
+added into tendrl for visualizing tendrl metrics. It will be using graphite
+as a database. This spec file address the required changes for all the tendrl
+components to support grafana for monitoring tendrl metrics.
+
+
+== Problem description
+
+* Currently tendrl has its own dashboard for visualizating the monitoring
+data. It does not support grafana. This document will provides the details
+about the required changes in every tendrl components to support grafana
+according to package prospective.
+
+
+== Use Cases
+
+Install tendrl packages and its depdendencies to support grafana integration
+
+== Proposed change
+
+* The Makefile, spec file, service file and other build related files used
+in the tendrl app should be updated to support grafana integration.
+* All the related jobs in tendrl-copr, jenkins should also updated.
+
+
+=== Alternatives
+
+None
+
+
+=== Data model impact:
+
+None
+
+
+=== Impacted Modules:
+
```

Comment:
I think that **Tendrl/tendrl-ansible impact** could be added there and issue of tendrl-ansible tracking this (https://github.com/Tendrl/tendrl-ansible/issues/18) could be linked.